### PR TITLE
280 Email text change

### PR DESCRIPTION
**Pull request by Jelout**
## Summary by CodeRabbit

- **Documentation**
	- Updated the training feedback email template to include a hyperlink to the FCOA full training manual for better reference and compliance.

### CODE REVIEW

File: resources/views/emails/approve-certificate.blade.php [hunk 1]
```diff
@@ -7,14 +7,15 @@
     Thank you for being part of the FCOA mission. If you had a good time and appreciated your coach then feel free to send him/her a message of appreciation.
     <br>
     <br>
-    In order to Evolve we like to ask a few questions that you may discuss with your coach or you may send a private message to the FCOA administration on whatsapp number +639174489590
+    In order to Evolve we like to ask a few questions that you may discuss with your coach or
```
Comment (sirking1991) [inline resources/views/emails/approve-certificate.blade.php:10]: @Jelout  I think `Evolve` is a typo, did you mean `evolve`?

**Review by sirking1991 — CHANGES_REQUESTED** [review]
possible typo

File: resources/views/emails/approve-certificate.blade.php (hunk 1 above)
Comment (Jelout) [reply resources/views/emails/approve-certificate.blade.php:10]: yes I did mean evolve. But I'm not sure how to change it now

Comment (sirking1991) [reply resources/views/emails/approve-certificate.blade.php:10]: just change it in your code, commit and push

Comment (Jelout) [reply resources/views/emails/approve-certificate.blade.php:10]: Oh I just made a new pull request for it, so this pull request can get deleted or closed. At the moment I can't see the changes I make in my local. So I should not make further pull requests until that issue is resolved.

**Comment by sirking1991** [comment]
Closing this PR

**Pull request closed without merging**